pkg/subgraph/src/schema.graphql

```diff
@@ -4,12 +4,11 @@ type CVStrategy @entity(immutable: false) {
   poolAmount: BigInt
   metadata: String
   registryCommunity: RegistryCommunity!
-  config: CVStrategyConfig
+  config: CVStrategyConfig!
```

Comment:
i see, you are chaging to not null, i normally dont like enforce it, because we can change the order and are using templates, but then you can discosider the last comments about config.